lib/elastomer_client/client/bulk.rb

```diff
@@ -280,9 +280,9 @@ def create(document, params)
       # params   - Parameters for the update action (as a Hash) (optional)
       #
       # Examples
-      #   update({"foo" => "bar"}, {:_id => 1}
-      #   update({"foo" => "bar"}, {:id => 1}
-      #   update("foo" => "bar", "_id" => 1)
+      #   update({"doc" => {"foo" => "bar"}}, {:_id => 1}
+      #   update({"doc" => {"foo" => "bar"}}, {:id => 1}
+      #   update({"doc" => {foo" => "bar"}}, "_id" => 1)
```

Comment:
```suggestion
      #   update({"doc" => {"foo" => "bar"}}, {:_id => 1})
      #   update({"doc" => {"foo" => "bar"}}, {:id => 1})
      #   update({"doc" => {"foo" => "bar"}}, "_id" => 1)
```
Minor updates, as all three seem to have slight syntax issues.

Reply:
Thanks for catching these @ndonewar ✨